ts/dx.all.d.ts

```diff
@@ -7894,17 +7929,17 @@ declare module DevExpress.viz.charts {
         /** Specifies options of the legend. */
         legend?: BaseChartLegend;
         /** A handler for the done event. */
-        onDone?: ((e: { component?: any, element?: any }) => any);
+        onDone?: ((e: { component?: DOMComponent, element?: DevExpress.core.dxElement, model?: any }) => any);
         /** A handler for the pointClick event. */
-        onPointClick?: ((e: { component?: any, element?: any, jQueryEvent?: JQueryEventObject, target?: basePointObject }) => any) | string;
+        onPointClick?: ((e: { component?: any, element?: any, jQueryEvent?: JQueryEventObject, event?: event, target?: basePointObject }) => any) | string;
```

Comment:
component: DOMComponent.
it would be great to use the DevExpress.viz.dxChart type